azure-pipelines.yml

```diff
@@ -81,43 +82,41 @@ jobs:
       maxParallel: 1
       matrix:
         open-jdk-8-linux:
-          imageName: 'MMSUbuntu20.04TLS'
+          IMAGE_TYPE: 'ubuntu-latest'
           JDK_DOWNLOAD_LINK: 'https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u322-b06/OpenJDK8U-jdk_x64_linux_hotspot_8u322b06.tar.gz'
           JAVA_VERSION: 'OpenJDK8U-jdk_x64_windows_hotspot_8u322b06'
           JDK_PATH: 'jdk8u322-b06'
           JAVA_VERSION_SPEC: '8'
         open-jdk-8-windows:
-          imageName: 'MMS2019TLS'
```

Comment:
Checking opinion of @eliaslopezgt as previously we changed this from `windows-2019` to `MMS2019TLS`
https://github.com/Azure/azure-functions-java-worker/pull/497

Reply:
This is fine since it is using it for the test job.